artwork/logo/coala-header.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <!-- Bootstrap CDN -->
+    <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-WskhaSGFgHYWDcbwN70/dfYBj47jz9qbsMId/iRN3ewGhXQFZCSftd1LZCfmhktB" crossorigin="anonymous">
+    <!-- Roboto Font CDN -->
+    <link href="https://fonts.googleapis.com/css?family=Roboto:100" rel="stylesheet">
+    <!-- Link the CSS file -->
+    <link rel="stylesheet" href="coala-header.css">
+    <title>Documentation</title>
+</head>
+<body>
+    <section class="background container">
+        <div class="row center">
+            <div class="col-md-3 col-sm-6  mx-auto">
```

Comment:
double spaces should be single

Reply:
As far as I can see, mx-auto is not needed in this context, can you explain why it is used?

Reply:
If you want to use mx-auto, you have to specify width of the node.
and also the child, you should use `position: relative`.